src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoOffhand.kt

```diff
@@ -95,8 +96,8 @@ object AutoOffhand : Module() {
             if (!transactionLog.containsValue(false)) movingTimer.reset(-175L) // If all the click packets were accepted then we reset the timer for next moving
         }
 
-        listener<SafeTickEvent>(1100) {
-            if (mc.player.isDead || !movingTimer.tick(200L, false)) return@listener // Delays 4 ticks by default
+        safeListener<TickEvent.ClientTickEvent>(1100) {
+            if (mc.player.isDead || !movingTimer.tick(200L, false)) return@safeListener // Delays 4 ticks by default
             if (!mc.player.inventory.itemStack.isEmpty) { // If player is holding an in inventory
```

Comment:
All usages such as `mc.player`, `mc.world`, `mc.connection`, `mc.playerController` in `safeTickListener` should be replaced with the non null ones in `SafeClientEvent`

Reply:
See code in ElytraFlight for example